process/smartContract/process.go

```diff
@@ -1374,6 +1393,26 @@ func (sc *scProcessor) DeploySmartContract(tx data.TransactionHandler, acntSnd s
 		return 0, err
 	}
 
+	sw := core.NewStopWatch()
+	sw.Start("deploy")
+
+	returnCode, err := sc.doDeploySmartContract(tx, acntSnd)
+	sw.Stop("deploy")
+	duration := sw.GetMeasurement("deploy")
+
+	if duration > executeDurationAlarmThreshold {
+		log.Debug(fmt.Sprintf("scProcessor.DeploySmartContract(): execution took > %s", duration), "sc", tx.GetRcvAddr(), "duration", duration, "returnCode", returnCode, "err", err, "data", string(tx.GetData()))
```

Comment:
executeDurationAlarmThreshold instead duration in %s, as the duration is displayed in "duration"